src/migrations/20230828103020-program-personnel-cleanup.js

```diff
@@ -0,0 +1,165 @@
+const {
+  prepMigration,
+  setAuditLoggingState,
+} = require('../lib/migration');
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface) {
+    await queryInterface.sequelize.transaction(async (transaction) => {
+      const sessionSig = __filename;
+      await prepMigration(queryInterface, transaction, sessionSig);
+
+      await queryInterface.sequelize.query(`
+
+        /* 1. Create a temp table of dedupe. */
+         DROP TABLE IF EXISTS "ProgramPersonnelToKeep";
+         CREATE TABLE "ProgramPersonnelToKeep" AS (
```

Comment:
This needs to be a temp table so it does not get an audit log. That way it will be dropped when the transaction is over